src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -225,23 +229,23 @@ public JsonClassInfo ElementClassInfo
             }
         }
 
-        public Type ElementType { get; set; }
+        public Type? ElementType { get; set; }
 
-        public JsonEnumerableConverter EnumerableConverter { get; private set; }
-        public JsonDictionaryConverter DictionaryConverter { get; private set; }
+        public JsonEnumerableConverter? EnumerableConverter { get; private set; }
+        public JsonDictionaryConverter? DictionaryConverter { get; private set; }
 
         // The escaped name passed to the writer.
         // Use a field here (not a property) to avoid value semantics.
         public JsonEncodedText? EscapedName;
 
-        public static TAttribute GetAttribute<TAttribute>(PropertyInfo propertyInfo) where TAttribute : Attribute
+        public static TAttribute? GetAttribute<TAttribute>(PropertyInfo? propertyInfo) where TAttribute : Attribute
         {
-            return (TAttribute)propertyInfo?.GetCustomAttribute(typeof(TAttribute), inherit: false);
+            return (TAttribute?)propertyInfo?.GetCustomAttribute(typeof(TAttribute), inherit: false);
```

Comment:
I think `propertyInfo` can never be null. Remove nullability on the parameter?

```suggestion
            return (TAttribute?)propertyInfo.GetCustomAttribute(typeof(TAttribute), inherit: false);
```

Reply:
sure